design-proposals/emts-scale-provisioning.md

```diff
@@ -0,0 +1,255 @@
+
+### Detach of Edge Nodes
+
+Once ENs are provisioned and ready to be shipped to the field to operate as standalone ENs, they should be detached from the local EMF orchestrator.
+An explicit detach would be needed if there are agents' configurations, certificates, credentials, etc. installed on ENs.
+However, **BM agents won't be configured and activated for EMT-S OS profiles**. Therefore, the only operation to detach ENs from the orchestrator is to delete them via northbound API.
+This is already supported via UI or CLI.
+
+### OXM profile of EMF deployment (EIM-S)
+
+The provisioning of EMT-S at scale will be driven by a local OXM orchestrator instance that only includes EIM components. 
+The main requirement of EIM standalone is to make it easy and resource-efficient for customers to deploy.
+
+For MVP, we will leverage the EIM Standalone profile. This is an EMF deployment profile that only deploys EIM with Observability and FPS components.
+Also, EIM won't include `infra-managers` for the OXM profile.
+
+EIM Standalone profile for minimal OXM warehouse deployments will be configured with EMT-Standalone OS profile only.
+
+Any further optimization to make EIM deployment fast and easy will be handled as part of the effort on the new installer (see [ADR](https://github.com/open-edge-platform/edge-manageability-framework/pull/290)).
+To give an example, further optimizations may include deploying EIM as a self-contained Helm chart (instead of ArgoCD appplication) and getting rid of FPS services that are
+not required (e.g., Argo, Istio, HA capabilities, etc.).
+
+### Rationale
+
+The current design proposal allows to easily support legacy PXE boot while keeping the current UX around EN pre-registration. It also lets us keep all the features that EIM currently supports,
+including logging, KPI instrumentation, observability, etc. The EN pre-registration also enables selectively configuring desired OS profile per EN and acts as
+admission control to prevent unwanted ENs being provisioned to EMF "magically". 
+
+The alternative considered was to use a standalone Tinkerbell deployment without the rest of the EIM stack. While it has advantage of a more lightweight deployment,
+it would completely change the UX as we would need to familiarize customers with Tinkerbell APIs (or have a custom CLI tool to help them manage Tinkerbell CRDs).
+With the current proposal we keep using the current UX, with possibility to use Bulk Import Tool to scale preregistration process and selectively choose OS profiles for Edge Nodes.
+
+Also, an alternative to using SMEE was considered to avoid divergence in security policies (HTTP vs. HTTPS for Provisioning Nginx).
+This direction requires more development effort, but is still left as future improvement. It's further elaborated in a separate [design proposal](https://github.com/open-edge-platform/edge-manageability-framework/pull/309).
+
+## Affected components and Teams
```

Comment:
@daniele-moro to add the role to avoid User accessing App and CO.

Reply:
We can use this group https://docs.openedgeplatform.intel.com/edge-manage-docs/main/shared/shared_iam_groups.html#project-id-host-manager-group, as long as `[<project-id>_im-rw](https://docs.openedgeplatform.intel.com/edge-manage-docs/main/shared/shared_iam_groups.html#project-id-im-rw)` role is there, we are ok from edgeinfra perspective.